next.config.js

```diff
@@ -15,7 +15,7 @@ const securityHeaders = [
       };` +
       `style-src 'self' fonts.googleapis.com 'unsafe-inline';` +
       `font-src fonts.gstatic.com;` +
-      `connect-src 'self' blockscout-index.changi.dfi.team ${
+      `connect-src 'self' blockscout-index.changi.dfi.team solc-bin.ethereum.org/linux-amd64/list.json raw.githubusercontent.com/blockscout/solc-bin/main/vyper.list.json blockscout.com 35.197.159.154:4000 ${
```

Comment:
We can replace `35.197.159.154:4000` with `blockscout-index.changi.dfi.team` right?
```suggestion
      `connect-src 'self' blockscout-index.changi.dfi.team solc-bin.ethereum.org/linux-amd64/list.json raw.githubusercontent.com/blockscout/solc-bin/main/vyper.list.json blockscout.com ${
```